src/internal/m365/collection/drive/helper_test.go

```diff
@@ -1053,11 +1053,11 @@ type deltaQuery struct {
 }
 
 func delta(
```

Comment:
thoughts on also updating `deltaWReset` to have the same set of parameters?

Reply:
oohh, good idea.